### ISSUE TITLE
API currently broken

### ISSUE THREAD

**Issue by m9brady** [m9brady/eodms-api-client#4]
**Describe the bug**
EODMS API was updated alongside a server version upgrade and has several changes in behaviour

**To Reproduce**
Try to order/download using this package

**Additional context**
**Searching:**
- working as expected

**Ordering:**
- "works" in that it submits an order that is accepted by EODMS
- As of 2022-04-06 14:45 Eastern no orders submitted by API get past the `SUBMITTED` stage
  - this behaviour occurs regardless if using this package or NRCan's
  - my _guess_ is that there is a server-side issue that is blocking processing
- workaround: use the EODMS web interface until API is patched

**Downloading:**
- Unable to query API by OrderId for order statuses
  - supplying the `?orderId=<num>` parameter used to provide only the scenes that belong to an order but now it always responds with the last-100-ordered scenes
  - this behaviour is a regression of a previous bug - see context/discussion in #3 
  - server-side issue 
- workaround: use the EODMS web interface until API is patched

**Comment by m9brady** [comment]
Pushed a band-aid fix for downloading in 410e3e91093f4662eebbb675d092a7d88fe7c3a6

Not planning to make a release as these changes will probably have to be retracted once the API is patched but users who want to try it out can install direct from github with:
```
pip install https://github.com/m9brady/eodms-api-client/archive/410e3e91093f4662eebbb675d092a7d88fe7c3a6.zip
```
The hash is specific to the aforementioned commit. Obviously this is at the user's discretion but it currently passes the "WORKS ON MY MACHINE" test for RCM/RS2

**this will not help you if your orders contain more than 100 images as issuing a GET to the Order endpoint for the purposes of checking order status appears to always return the last-100-ordered images**

Ordering by API appears to still not function as expected as of 2022-04-07T11:27:00-04:00

**Comment by m9brady** [comment]
Marking this as resolved as the package is in a "working" state. For more details, consult the recent release: https://github.com/m9brady/eodms-api-client/releases/tag/v1.1.5
